README.md

```diff
@@ -2,4 +2,18 @@
 
 This is the repo for sketching out the initial deployment of the msc course to epimodels.dide.ic.ac.uk; once this is working we'll rewrite it to generally support a set of configurations.
 
-* we need to support multiple configurations easily; for now I've put the configuration in config/msc-idm-2022 to reflect one way we might achieve this practically
+* we need to support multiple configurations easily; for now I've put the configuration in `config/msc-idm-2022` to reflect one way we might achieve this practically
+* there's another configuration now at `config/demo` - this time from a submodule!
```

Comment:
Looks like they're both in submodules?

Reply:
I knew there was an out of date comment somewhere!